templates/CentOS/6/etc/fail2ban/jail.conf.epp

```diff
@@ -322,41 +322,41 @@ logpath = %(lighttpd_error_log)s
 
 [roundcube-auth]
 enabled = <%= "roundcube-auth" in $fail2ban::jails %>
-port     = http,https
+port     = <%= lookup('fail2ban::jails_config.roundcube-auth.port', String, undef, 'http,https') %>
 logpath  = logpath = %(roundcube_errors_log)s
```

Comment:
why does this say `logpath = logpath = … `

Reply:
Thanks, that doesn't seem to be right.